wagtail/core/rich_text/rewriters.py

```diff
@@ -56,12 +56,28 @@ def replace_tag(self, match):
         try:
             link_type = attrs['linktype']
         except KeyError:
-            # return ordinary links without a linktype unchanged
-            return match.group(0)
+            link_type = None
+            href = attrs.get('href', None)
+            if href:
+                # From href attribute we tried to detect only the linktypes that we
+                # currently support (`external` and `email`. `page` have default handler)
+                # from our link chooser.
+                if href.startswith(('http', 'https')):
```

Comment:
I'll make a small change to add the `:` there so we can be sure to match a protocol. It shouldn't make any difference with the current link chooser's validation, but there might be people using custom link choosers that have invalid links.